admin_manual/configuration_server/android_deep_link_handling.rst

```diff
@@ -0,0 +1,49 @@
+Deep Link Handling
+====================================
+
+Deep linking in Android allows your application to be launched directly from a URL, 
+making it easier for users to navigate to specific content within your app. 
+Starting from Android 12, handling deep links requires additional configuration 
+using an assetlinks.json file to ensure the app and the host domain are properly 
+associated.
+
+Deep Linking in Android 11 and Below
+====================================
+For Android 11 and below, deep linking is straightforward and does not require additional 
+configuration beyond the usual manifest settings.
+
+Deep Linking in Android 12 and Above
+====================================
+For Android 12 and above, an additional configuration step is required to verify the 
+relationship between your app and the host domain using the assetlinks.json file.
+
+Creating assetlinks.json:
+-------------------------
+Create a file named assetlinks.json and host it in the .well-known directory of 
+your website (e.g., https://www.cloud.example.com/.well-known/assetlinks.json).
+
+Example assetlinks.json::
+
+    [
+      {
+        "relation": ["delegate_permission/common.handle_all_urls"],
+        "target": {
+          "namespace": "android_app",
+          "package_name": "com.cloud.example.nextcloud",
+          "sha256_cert_fingerprints": [
+            "APP_SHA256_FINGERPRINT"
+          ]
+        }
+      }
+    ]
+
+Replace APP_SHA256_FINGERPRINT with the SHA-256 fingerprint of your app’s 
+signing certificate.
```

Comment:
Oh, this is only applicable to branded apps, right? 

This either needs a *big* caveat at the top of the chapter or maybe it shouldn't be included here since it's not applicable to community users... so maybe it only belongs in the customer docs in https://portal.nextcloud.com somewhere?

Reply:
@alperozturk96 this also works for our official app, right?
So maybe add the sha256 of our app here.
And as Josh suggested, add a portal entry how this needs to be changed for branded apps.

Reply:
@tobiasKaminsky How can I add portal entry?

Reply:
https://github.com/nextcloud-gmbh/portal.nextcloud.com